TKGame/Level Editor Content/Stages/auto_saved_stage_data_1.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
You probably don't need to include this file in the commits, unless you need some kind of custom layout.